README.md

```diff
@@ -470,16 +475,16 @@ IEEE Robotics and Automation Letters (RA-L), 2:2(632-639), 2017.
 - [Mueggler et. al. IJRR 2017](#Mueggler17ijrr).  
 *The Event-Camera Dataset and Simulator: Event-based Data for Pose Estimation, Visual Odometry, and SLAM.*
 
-<a name="visual-inertial"></a>
+<a name="f"></a>
```

Comment:
Please revert this change.  It breaks the link in the table of contents.

Reply:
Oops, this was unintended, I've reverted the change.